src/telemetry.ts

```diff
@@ -11,17 +11,19 @@ import {
     NativeMouseCommandTelemetry,
     Telemetry,
     VSCodeNativeTelemetry
-} from '../platform/common/constants';
-import { CheckboxState, EventName, PlatformErrors, SliceOperationSource } from './constants';
-import { DebuggingTelemetry } from '../kernels/debugger/constants';
-import { EnvironmentType, PythonEnvironment } from '../platform/pythonEnvironments/info';
-import { TelemetryErrorProperties, ErrorCategory } from '../platform/errors/types';
-import { ExportFormat } from '../notebooks/export/types';
-import { InterruptResult, KernelConnectionMetadata, KernelInterpreterDependencyResponse } from '../kernels/types';
-import { IExportedKernelService } from '../standalone/api/extension';
-import { PreferredKernelExactMatchReason } from '../notebooks/controllers/notebookControllerManager';
-import { SelectJupyterUriCommandSource } from '../kernels/jupyter/serverSelector';
-import { TerminalShellType } from '../platform/terminals/types';
+} from './platform/common/constants';
+import { CheckboxState, EventName, PlatformErrors, SliceOperationSource } from './platform/telemetry/constants';
+import { DebuggingTelemetry } from './kernels/debugger/constants';
+import { EnvironmentType, PythonEnvironment } from './platform/pythonEnvironments/info';
+import { TelemetryErrorProperties, ErrorCategory } from './platform/errors/types';
+import { ExportFormat } from './notebooks/export/types';
+import { InterruptResult, KernelConnectionMetadata, KernelInterpreterDependencyResponse } from './kernels/types';
+import { IExportedKernelService } from './standalone/api/extension';
+import { PreferredKernelExactMatchReason } from './notebooks/controllers/notebookControllerManager';
+import { SelectJupyterUriCommandSource } from './kernels/jupyter/serverSelector';
+import { TerminalShellType } from './platform/terminals/types';
+
+export * from './platform/telemetry/index';
```

Comment:
We would move some of the functions into this file from `platform/telemetry/index` instead of re-export in following PRs.